config/locales/views/main/en.yml

```diff
@@ -66,7 +66,6 @@ en:
           forem: Forem
           opensource: open source
           dev: DEV
-        desc: "%{community} – %{desc}"
```

Comment:
That's the translation I mentioned earlier https://github.com/forem/forem/pull/15828/files#r772286736